src/tutorials/lua_basics_lightbulb.md

```diff
@@ -25,103 +25,184 @@ This involves turning on and off a light switch to illuminate a light bulb.
+4. Open the contents of the **Lightbulb & Switch** template by clicking the drop down arrow to the left of its name in the Hierarchy.
+5. Open the **Project Content** window and open the **My Scripts** category. Drag the `LightSwitchToggle` from **Project Content** > **My Scripts** into the **Lightbulb & Switch** template group.
 
-    !!! info
-        It's important to save your scripts often so you don't lose work in the event of a problem. Use ++Ctrl+Shift+S++ to save all scripts when you save your project.
+!["Create Script"](../img/LightBulb/CreateScript.png){: .image-frame .center loading="lazy" }
+!["Project Content"](../img/LightBulb/ProjectContent.png){: .image-frame .center loading="lazy" }
 
-### Deinstance the Lightbulb & Switch Template
+!!! info
+    It is important to save your scripts often so you don't lose work in the event of a problem. Use ++Ctrl+Shift+S++ to save all scripts and your entire project.
 
-To make changes to the template, we first need to **deinstance** it.
+### Create and Customize a Light
 
-Right click on the **Light bulb & Switch** template in the **Hierarchy** and select **Deinstance This Object**.
+1. Open the **Lighting** category of the **Core Content** window.
+2. Find the **Point Light** and drag it into the hierarchy, under **Lightbulb & Switch** > **Lightbulb** > **Bulb**.
+3. With the **Point Light** selected in the **Hierarchy**, open the **Properties Window**.
+4. Lower the **Intensity** property until the light looks natural in your scene.
+5. In the **Lighting** section of **Properties**, check the box next to **Advanced Settings** to see more options to customize the light.
+6. Check the box for **Use Temperature**, and lower the **Temperature** property to create a warm glow color to the light.
+7. Since we want the light to be off by default, set the **Visibility** property to **Force Off**.
 
-!["Right Click Content Menu"](../img/LightBulb/image8.png){: .image-frame .center loading="lazy" }
+!["Point Light Properties"](../img/LightBulb/lightProperties.png "Point Light in the Properties window."){: .image-frame .center loading="lazy" }
 
-The template and objects in the template will change from **blue**{: style="color: var(--core-color-templetized)"} to **teal**{: style="color: var(--core-color-deinstanced)"}. This color change means that the template is now editable.
+### Find the Trigger
 
-Deinstancing allows you to make changes to a particular copy of a template without changing the template itself. You can learn more about how this works in our [template reference](template_reference.md).
+To create in-game interactions in Core, use a **Trigger**. A trigger defines the area and checks if any player enters, leaves, or presses the interact key, ++F++, near it.
 
-### Add the Script to the Template
+The **Lightbulb & Switch** template includes a trigger, but you can add trigger objects to a project from the **Core Content** window, in the **Gameplay Objects** section.
```

Comment:
I just checked and it does not look like it comes with a trigger

Reply:
The updated template does/will have a Trigger in it.

Reply:
Added Steps to Adding Trigger to the Template